e2e_test/iceberg/test_case/no_partition_append_only.toml

```diff
@@ -13,24 +13,27 @@ init_sqls = [
     v_date date,
     v_timestamp timestamp,
     v_ts_ntz timestamp_ntz,
-    v_decimal decimal(10,5)
+    v_decimal decimal(10,5),
+    v_map map<int, int>,
+    v_array array<int>,
+    v_struct struct<a:int, b:int>
```

Comment:
Have you ever tried nesting a nested type inside another nested type?

Reply:
Just tried, and found a new icelake bug. It will fail to parse table metadata 🥵

Reply:
Let's fix this later